module/zfs/vdev.c

```diff
@@ -264,6 +269,12 @@ vdev_default_asize(vdev_t *vd, uint64_t psize)
 	return (asize);
 }
 
+uint64_t
+vdev_default_min_asize(vdev_t *vd)
+{
+	return (vd->vdev_min_asize);
+}
```

Comment:
The existing ops were a little inconsistent about this same thing.  The new ops were implemented gradually on a case-by-case basis and for each one I opted for what felt cleanest bases on some simple rules.  There may be some exceptions but as a general rule `NULL` was used when the default behavior was to do nothing, and a `default_op` was used when there was a common action/calculation needed by more than one vdev type.  I think it could make sense to revisit this in a separate PR and establish a convention for them all.